README.md

```diff
@@ -47,6 +47,91 @@ If you want to contribute to Victoire frontend, please read this [contribution g
 
 To do so, follow the [setup guide](doc/setup.md).
 
+## Setup Victoire for development
```

Comment:
I'd rather to move this section to a `doc/setup-for-contributor.md` file to be more consistent with `doc/setup.md`